data/ecosystems/b/bnb-chain.toml

```diff
@@ -508,18 +519,27 @@ sub_ecosystems = [
 github_organizations = [
   "https://github.com/bnb-chain",
   "https://github.com/daohuntapp",
+  "https://github.com/CookieCommunity",
+  "https://github.com/CratosTokenOfficial",
   "https://github.com/defi-org-code",
   "https://github.com/dexe-network",
   "https://github.com/envoys-lab",
   "https://github.com/ERC721RA",
   "https://github.com/Everest-Option-Exchange-Team",
   "https://github.com/Fexon-Finance",
+  "https://github.com/grindery-io",
+  "https://github.com/imemessenger",
   "https://github.com/label-platform",
+  "https://github.com/Igocoin",
   "https://github.com/liminal-market",
+  "https://github.com/locus-finance",
   "https://github.com/LuckyMachines",
   "https://github.com/Marketplace-of-the-Multiverse",
   "https://github.com/NFTin-App",
   "https://github.com/pandora-nft",
+  "https://github.com/reclaimprotocol",
+  "https://github.com/stabilityworld",
+  "https://github.com/vooi-app",
```

Comment:
Since you added the sub ecosystems above, you don't also need to add the github orgs here.

Reply:
@jubos please check the changes